openassessment/xblock/submission_mixin.py

```diff
@@ -171,6 +171,13 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
                 )
                 if answer_too_long:
                     status_tag = 'EANSWERLENGTH'
+                    max_size = f"({int(api.Submission.MAXSIZE / 1024)} KB)"
+                    base_error = self._("Response exceeds maximum allowed size.")
+                    extra_info = self._(
+                        "Note: if you have a spellcheck or grammar check browser extension, "
+                        "try disabling, reloading, and reentering your response before submitting."
+                    )
+                    status_text = f"{base_error} {max_size} {extra_info}"
```

Comment:
This code path doesn't log the error. Is that something we want to have logged for our tracking?

Reply:
Yep, missed that